README.md

```diff
@@ -154,7 +154,25 @@ If information about the specific commit that is being deployed is available at
     --deploy_end_time "2021-07-20T18:08:42.024Z"
 ```
 
-**Option 2: **
+Example Usage: 
+
+```sh
+./faros_event.sh CD -k "<faros_api_key>" \
+    --commit "GitHub://faros-ai/faros-events-api/4414ad2b3b13b17055171678437a92e5d788cad1" \
+    --deploy "Spinnaker://faros-events-api/Prod/4414ad2b3b13b17055171678437a92e5d788cad1_156" \
+    --deploy_status "Success" \
+    --deploy_start_time "2021-07-20T18:05:46.019Z" \
+    --deploy_end_time "2021-07-20T18:08:42.024Z"
+```
```

Comment:
Do you think having both the example usage and the above event with field placeholders is worth it or should we just have your example usage?